cdc/sink/codec/avro.go

```diff
@@ -259,9 +267,15 @@ func getAvroDataTypeNameMysql(tp byte) (string, error) {
 	case mysql.TypeVarchar, mysql.TypeString, mysql.TypeVarString:
 		return "string", nil
 	case mysql.TypeDate, mysql.TypeDatetime, mysql.TypeTimestamp:
-		return "long", nil
+		return avroLogicalType{
+			Type:        "long",
+			LogicalType: "timestamp-millis",
```

Comment:
Recommend to define `type LogicalType string` and use const variables like
`TimestampMillis LogicalType = "timestamp-millis"`